dbms/programs/client/Client.cpp

```diff
@@ -61,7 +61,7 @@
 #include <AggregateFunctions/registerAggregateFunctions.h>
 
 #if USE_READLINE
-#include "Suggest.h"
+#include "Suggest.h" // Y_IGNORE
```

Comment:
?

Reply:
arcadia build without USE_READLINE and Suggest.h not copied to arcadia.
And on ya make run file "Suggest.h" is missing

Reply:
All files should be present in Arcadia.

Reply:
Will be soon. Now only needed for compilation copied.